JavaAssignments/src/saket/Addition.java

```diff
@@ -0,0 +1,32 @@
+package saket;
+
+public class Addition {
+
+	void add(int a, int b) {
+		int c = a + b;
+
+		System.out.println(c);
+
+	}
+
+	void add(float a, float b) {
+		double c = a + b;
+
+		System.out.println(c);
+
+	}
+
+	void add(String a, String b) {
+		String c = a + b;
```

Comment:
Here it concat the a and b.  It not give me sum 
Right? 
Need to parse value n then you do sum